### PR TITLE
Warning for when accesskey attribute is used

### LINKED ISSUE #49: Show warning if element uses accesskeys
### Describe the new a11y feature or project enhancement

Screen readers and other assistive technologies use shortcuts and keyboard combinations to facilitate their use. Access keys could interfere and conflict with the AT behavior and should be avoided.

### Describe the solution you'd like

Show a warning if an element has an access key. As it is an HTML attribute (`accesskey`), the elements with access keys are easily selectable via CSS:

```css
[accesskey] {
  /* warning */
}
```

### Link(s)

- [WebAIM Keyboard Accessibility: Accesskey](https://webaim.org/techniques/keyboard/accesskey)
- [MDN Global Attributes: Accesskey](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey)

### PR DESCRIPTION
**Pull request by azhsetiawan**
## Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
Add highlight and warning message on element with `accesskey` attribute.
Resolves #49 

## Link(s)
[WebAIM: Keyboard Accessibility - Accesskey](https://webaim.org/techniques/keyboard/accesskey)
[accesskey - HTML: HyperText Markup Language | MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey)

## Screenshot(s)
image

## Checklist:

- [x] I have thoroughly read the [CONTRIBUTING](../CONTRIBUTING.md) guidelines.
- [x] I understand my pull request will be thoroughly reviewed at high detail.
- [x] I understand the work in my pull request will **only** be available in the next version release of Checka11y.css and not in the current version release.
- [x] I confirm the work in this pull request is valid according to my findings and is not something for anything personal.
- [x] I have updated the [README](../README.md) where and if applicable (still put an `x` if you have considered this but thought there was nothing to add or modify).
- [x] I have added myself to the `contributors` section in `package.json` (still put an `x` if you have considered this but decided not to add yourself).
- [x] I have checked I have **not** committed any **accidental** files.
- [x] I have tested all the main modern browsers (I.e. Chrome, Firefox, Edge, Safari - please leave this unchecked if there were any browsers listed you could not test and list them in the [help](#help) section with details why you couldn't test that browser)
- [x] All new and existing a11y checks still work correctly (compare your local `test/index.html` to the `test/index.html` in the `master` branch).

## Help

Tested on Firefox, Chrome, and Safari on macOS, could not test in any Windows browser.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ A list of every a11y concern Checka11y.css will check for and highlight:
 - Checks headings have meaningful content:
   - They cannot be empty
   - They are reachable by assistive technologies (no `aria-hidden`)
+- Checks usage `accesskey` attribute in elements should be avoided
```
Comment (jackdomleo7) [inline README.md:73]: I appreciate the PR I'm about to mention was being worked on at the same time as you were working on this, but could you have a read through #47 and change this accordingly? Basically, this line needs relocating into `features.md`. 😊

File: checka11y.css [hunk 2]
```diff
@@ -472,3 +472,15 @@ h6[aria-hidden]::after {
...
+
+/**
+* Accesskey attribute could interfere and conflict with screen readers and assistive technologies
+*/
+[accesskey] {
+  border: var(--checka11y-error-border-width, 5px) solid var(--checka11y-error-color, red) !important;
+}
+
+[accesskey]::after {
+  content: "ERROR: accesskey attribute could interfere and conflict with screen readers and assistive technologies" !important;
```
Comment (jackdomleo7) [inline checka11y.css:484]: @alvaromontoro Did you recommend this be a warning, rather than an error? 🙂

**Review by jackdomleo7 — CHANGES_REQUESTED** [review]
Very nice work, thank you! Just a few changes and a question for @alvaromontoro. Also, could you merge `master` into your branch to ensure it's up to date.

File: checka11y.css (hunk 2 above)
Comment (alvaromontoro) [reply checka11y.css:484]: It's a warning. Accesskey is not bad, just if the selected keys match the screen reader keys.

File: README.md (hunk 1 above)
Comment (azhsetiawan) [reply README.md:73]: I missed to read PR #47 
I'll update this PR then

**Review by jackdomleo7 — APPROVED** [review]
Nice! Thank you!

If you get chance, could you update the screenshot in the PR description? 😊

**Comment by azhsetiawan** [comment]
You're welcome! glad to contribute in this repo
I've updated screenshot in PR description

**Pull request merged**